### DISCUSSION TITLE
Find Release window: Sometimes, Master Releases do not open/expand

### DISCUSSION THREAD

**Discussion by gitgetgot** [ghDaYuYu/foo_discogs#60] [Spam/Abuse]
Version: 1.0.4
Compiled: Aug 19 2021

Doubleclicking on a Master Release opens its. 
But not in all cases and I have no idea why -on some Master Releases- the + will not expand.

Quite difficult to reproduce.
In the animated Gif, you see me doubleclicking like hell on all of the Master Releases, but only some will open.

After a restart of fb2k everything is working as expected, but then suddenly, the bug comes back again...

No idea how to trigger that bug... sorry. (Maybe its the text-filter...?)

![doubleclick does not work](https://user-images.githubusercontent.com/974980/130296953-b2a30e3f-777f-4ac0-8a84-3035db3fe682.gif)

**Comment by ghuDaYuYu** [comment]
Have you checked (removing the filter and expanding) that there is content inside those nodes (refusing to expand) meeting the search criteria 'st'. All maSTers will meet the criteria, but only those containing releases that also meet the criteria will expand. Not very intuitive I know.

**Comment by gitgetgot** [comment]
Yes I have checked that, with the old component, they open as expected and have content.

Normally, the releases under a MR have the same name, so they should also be shown after expanding.

In rare cases, some releases have different names than the MR's name, and I don't expect them to show up then.

**Comment by ghuDaYuYu** [comment]
Really? Sorry, I don´t believe it : )
Probe it! Post again the releases in your first gif without the filter. Otherwise, I would have to say that you are simply lying.

**Comment by gitgetgot** [comment]
You're right, I was simply lying!

...and after realising your capital ST in "maSTers", I understand, that the filter filters the word "masters" too.
Not very intuitive, as you said, in deed!

I remember now, why I changed "Master" to "M" in my setup years ago....

Thanks and sorry for the hassle!

**Reply by ghuDaYuYu** [reply]
That sound better my friend, we all make mistakes.
Hope we all learn a lesson from the hassles!
Regards

**Comment by ghuDaYuYu** [comment]
Think I had one of those 'aha' moments. It could be cool to have those masters grayed out.
Well, and sorry, too gitgetgot I may have been just a wee bit too harsh.
Till next time
